tools/system_message.toit

```diff
@@ -37,35 +40,67 @@ usage prog_name:
       $prog_name <snapshot>
       $prog_name <snapshot> <system message or heap_dump file>
       $prog_name <snapshot> -b <base64-encoded-ubjson>
+      $prog_name [--snapshot|-s <snapshot>] --message|-m <base64-encoded-ubjson>
       # Eg snapshot file can be toit.run.snapshot
 
     If no system-message file is given, the stack trace is read from stdin."""
   exit 1
 
 main args:
   prog_name := "system_message"
```

Comment:
Can't we just clean this up? We use tools/system_message in very few places, so it feels nicer just to get that sorted out. I'd also like to rename it to `decode_system_message` or something like that.